sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/EncryptParameters.cs

```diff
@@ -112,6 +112,16 @@ public static EncryptParameters A192CbcParameters(byte[] plaintext, byte[] iv =
         public static EncryptParameters A256CbcParameters(byte[] plaintext, byte[] iv = null) =>
             new EncryptParameters(EncryptionAlgorithm.A256Cbc, plaintext, iv, null);
 
+        /// <summary>
+        /// Creates an instance of the <see cref="EncryptParameters"/> class for the <see cref="EncryptionAlgorithm.CkmAesKeyWrap"/> encryption algorithm.
+        /// </summary>
+        /// <param name="plaintext">The plaintext to encrypt.</param>
```

Comment:
```suggestion
        /// <param name="plaintext">The text to encrypt.</param>
```
There's no reason to use "plain" here for the description.  (param name is good, though)